### DISCUSSION TITLE
如何让其他可以拖拽的元素在拖动到Tree节点上时，Tree节点可以处理拖拽事件

### DISCUSSION THREAD

**Discussion by ChenKun1997** [ant-design/ant-design#45246] [Q&A]
我的需求是表格的某一行拖拽到Tree的某个节点上时，需要根据节点的数据和表格的数据调一个接口，我现在是使用原生事件让表格的一行可拖拽，但是拖拽到Tree的节点时是没有任何反应的，不会有样式的反馈也不会触发Tree组件拖拽相关的事件，我需要怎么做呢
![image](https://github.com/ant-design/ant-design/assets/24249578/81e13be7-4c83-4064-9748-e0be858b4daa)

**Comment by ChenKun1997** [comment]
有大佬帮忙看下吗，如果是原生事件肯定是会触发onDragOver，onDrop等这些事件，但是Tree组件的这些事件都是内部封装好的，有什么办法可以让我的表格Row拖拽到TreeNode时触发这些事件呢

**Reply by xiaogonggong-w** [reply]
同求
